docs/book/requirements.rst

```diff
@@ -0,0 +1,88 @@
+.. index::
+    single: System Requirements
```

Comment:
Here's the [Sphinx documentation about it](http://www.sphinx-doc.org/en/stable/markup/misc.html#directive-index), but I can't understand it anyway :tada: